File: Lab5/golang/main.go
package main

import "fmt"

func main() {
	var count = 0
	for i := 1000; i < 10000; i++ {
		if i%101 == 0 {
			count++
			fmt.Println(count, ": ", i)
		}
	}

	fmt.Println("Кількість шуканих чисел:", count)
}

File: Lab5/javaScript/main.js
function check() {
   let count = 0;
   for (let i=1000;i<10000;i++){
            if (i%101 === 0){
                count++;
                console.log(count,": ",i);
            }
   }
    document.getElementById("result").innerText = "Кількість шуканих чисел: "+count;
}